model-server/README.md

```diff
@@ -0,0 +1,17 @@
+# `model-server` start helper
+
+This 'component' is not a real component but rather a start helper - the `model-server` is started using the gradle application plugin.
+
+To start the `model-server`, simply call the following in the repository root:
+
+```
+./gradlew model-server:run --args="-inmemory -dumpin courses.modelserver.dump"
```

Comment:
Maybe we can add these arguments as defaults to make calling easier for the common case?

Reply:
Unfortunately - as far as I could research - impossible with gradle at the moment.

Reply:
This is obsolete now with the discussions below.